app/pipeline/chat_gpt_wrapper_pipeline.py

```
@@ -0,0 +1,93 @@
+import logging
+from typing import List, Optional
+
+from langchain_core.prompts import (
+    ChatPromptTemplate,
+)
+from app.common.pyris_message import IrisMessageRole, PyrisMessage
+from app.domain.chat.exercise_chat.exercise_chat_pipeline_execution_dto import (
+    ExerciseChatPipelineExecutionDTO,
+)
+from app.domain.data.text_message_content_dto import TextMessageContentDTO
```

Ruff (0.8.2)

`app.domain.data.text_message_content_dto.TextMessageContentDTO` imported but unused

Remove unused import: `app.domain.data.text_message_content_dto.TextMessageContentDTO`

(F401)